src/sql/base/browser/ui/panel/tab.component.ts

```diff
@@ -29,19 +29,22 @@ export class TabComponent implements OnDestroy {
 	@Input() public identifier: string;
 	@Input() private visibilityType: 'if' | 'visibility' = 'if';
 	private rendered = false;
+	private destroyed: boolean = false;;
 
 	constructor(
 		@Inject(forwardRef(() => ChangeDetectorRef)) private _cd: ChangeDetectorRef
 	) { }
 
 	public set active(val: boolean) {
-		this._active = val;
-		if (this.active) {
-			this.rendered = true;
-		}
-		this._cd.detectChanges();
-		if (this.active && this._child) {
-			this._child.layout();
+		if (!this.destroyed) {
```

Comment:
There's a couple cases where rapidly switching the dashboard context caused crashes - events get sent after the object is destroyed, and then using this._cd fails. We should really have a different bug for this and be systematic.